07-usage-case-studies.md

```diff
@@ -350,3 +350,20 @@ available at https://github.com/applied-math/demos.
   demonstrating the paths of Atlantic hurricane tracks from 1950-2012
   with coloring demonstrating category of storm.](images/hurricanes.png
   "Image frome https://github.com/applied-math/demos")
+
+
+## Riemann Problems and Jupyter Solutions
+
+This book should be completed in the next few months, and the current state can be viewed at http://www.clawpack.org/riemann_book/index.html.
+
+We first envisioned this project as a teaching aid to interactively illustrate difficult concepts for a graduate course in numerical methods for conservation laws. These are physical laws in the form of first-order hyperbolic partial differential equations that arise in wave propagation applications such as fluid dynamics, traffic flow, water waves (like tsunamis), and electromagnetic waves among others. The Riemann problem corresponds to a conservation law with a piecewise constant initial condition such that the problem is relatively simple to solve, while still capturing the characteristic dynamics of the conservation law. Due to its discrete nature, its simplicity and its capability to encode the dynamics, the Riemann problem is the key ingredient in modern numerical methods for conservation laws.
+
+The project naturally evolved into having different Jupyter notebooks, each aimed to solve the corresponding Riemann problem for each of the relevant applications such as acoustics, shallow-water equations, and Euler equations, for example, with the idea of compiling these notebooks into an interactive book where each chapter corresponds to a Riemann problem for a specific application. However, we also wanted to use our book to encourage the reading and publication of interactive notebooks. Therefore, we decided that our book should also have a printed and an HTML version in addition to notebooks available for downloading, in order to attract more readers and to encourage more authors and publishers to explore this interactive platform. 
```

Comment:
The project naturally evolved into ~~having different~~ Jupyter notebooks, each aimed to solve the corresponding Riemann problem for ~~each of the~~ a relevant application~~s~~ such as acoustics, shallow-water equations, and Euler equations.~~, for example, with~~ ~~the idea of compiling~~ The project evolved into compiling these notebooks into an interactive book where each chapter corresponds to a Riemann problem for a specific application. ~~However, w~~We also wanted to use our book to encourage the reading and publication of interactive notebooks. Therefore, we decided that our book should also have a printed and an HTML version in addition to notebooks available for downloading, ~~in order~~ to attract more readers and to encourage more authors and publishers to explore this interactive platform.